src/base/utils/net.h

```diff
@@ -29,12 +29,15 @@
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
 class QString;
+class QHostAddress;
 
 namespace Utils
 {
     namespace Net
     {
         bool isValidIP(const QString &ip);
+        bool isLoopbackAddress(const QHostAddress &clientAddress);
+        bool isLANAddress(const QHostAddress &clientAddress);
```

Comment:
`clientAddress` -> `addr`
there is no concept of "client" in these 2 helper functions